src/modules/commander/PreflightCheck.cpp

```diff
@@ -530,7 +530,7 @@ static bool ekf2Check(orb_advert_t *mavlink_log_pub, vehicle_status_s &vehicle_s
 
 	if (status.hgt_test_ratio > test_limit) {
 		if (report_fail) {
-			mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: EKF HGT ERROR");
+			mavlink_log_critical(mavlink_log_pub, "Preflight Fail: EKF Height Error");
```

Comment:
This comment applies to all the messages that contain "EKF":
1) the average user has no clue what an EKF is.  They are very confused by these messages.
2) the term "EKF" doesn't add any helpful information to any of these messages anyway. 
3) Suggest: "Preflight Fail: Height estimate error"